ert3/data/__init__.py

```diff
@@ -4,12 +4,14 @@
 from ert3.data._record import RecordTransmitter
 from ert3.data._record import SharedDiskRecordTransmitter
 from ert3.data._record import InMemoryRecordTransmitter
+from ert3.data._record import RecordType
```

Comment:
RecordType is becoming an overloaded term. It does not mean the same thing in ert-storage and ert3, which I believe might become an issue.

I think also that record type should be something that exists on all records (EnsembleRecord, MultiEnsembleRecord, Record)

Reply:
I completely agree and we need to sit down and have a discussion on what a record is to be. Especially a numerical one! This has been raised earlier as well :) But, it also goes far beyond the scope of this PR and will probably induce changes in ert, ert-storage and webviz-ert alike.

Reply:
When it comes to record type I agree that it makes sense to propagate that information into the ensemble record (which is why it is currently listed under feature work in the PR description - which implies that I will make an issue on it). However, I'm not sure whether we want to require uniform record type over a multi ensemble record...

Reply:
You might be right, but for now I would rename it to `data_format` or similar. Let us not get the whole "for some definition of..." this early in the game

Reply:
Notice that I did not introduce this now. It was implemented "a long time back". I just made the enum class publicly available...